src/asana/helpers.py

```diff
@@ -76,14 +76,9 @@ def default_due_date_str(reference_datetime: datetime = None) -> str:
 
 def _task_status_from_pull_request(pull_request: PullRequest) -> str:
     if not pull_request.closed():
```

Comment:
I like this format better! One nit: can you switch them around for easier readability? (rather than first condition being "not closed", have the first condition be `if pull_request.closed()`